### PR TITLE
LPD-17480

### PR DESCRIPTION
**Pull request by rijema**

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:relevant

**Comment by rijema** [comment]
ci:test:stable

**Comment by rijema** [comment]
ci:test:objects

**Comment by rijema** [comment]
ci:test:sf

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java [hunk 1]
```diff
@@ -310,7 +318,22 @@ public void testSendNotification() throws Exception {
 			false,
 			StringBundler.concat(
 				user1.getEmailAddress(), StringPool.COMMA,
-				user2.getEmailAddress()));
+				user2.getEmailAddress()),
+			ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		// Multiples emails for each main recipient with a ";" separator
```
Comment (pedro-oliveira446) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:324]: Here we need to put the comments in alphabetical order. I'm not sure that would be the ideal comment either.

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java [hunk 2]
```diff
@@ -310,7 +318,22 @@ public void testSendNotification() throws Exception {
...
+		// to a local Object Entry
+
+		_testSendNotification(
```
[the remark this answers is not in the document]
Comment (rijema) [reply modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:327]: Thanks, @pedro-oliveira446 . This code was removed.

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java (hunk 1 above)
Comment (rijema) [reply modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:324]: Thanks for reminding me of this!

**Comment by rijema** [comment]
Opening a new PR with the changes from a newer branch. Thx.

**Pull request closed without merging**